trackers/javascript-tracker/tracker.test.config.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2021 Snowplow Analytics Ltd, 2010 Anthon Pang
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of the copyright holder nor the names of its
+ *    contributors may be used to endorse or promote products derived from
+ *    this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+export const plugins = {
+  performanceTiming: true,
+  gaCookies: true,
+  geolocation: true,
+  optimizelyX: true,
+  clientHints: true,
+  consent: true,
+  linkClickTracking: true,
+  formTracking: true,
+  errorTracking: true,
+  timezone: true,
+  ecommerce: true,
+  enhancedEcommerce: true,
+  adTracking: true,
+  siteTracking: true,
+  optimizely: false,
+  browserFeatures: false,
+  mediaTracking: false,
```

Comment:
I think it should be true here, so your tests build correctly, no? 

Looks like `rollup.config.test.js` isn't configured correctly and is using `tracker.config.ts` (because thats the default). You should add the `alias` plugin to `rollup.config.test.js` just like we do to build the `lite` version of the tracker in the main build.

https://github.com/snowplow/snowplow-javascript-tracker/blob/13183fc827fd4b5ce24e451357ae33c0ed442155/trackers/javascript-tracker/rollup.config.js#L68